code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -1,4 +1,5 @@
 /mob/living/carbon/human/var/cpr_cooldown
+/mob/living/carbon/human/var/cpr_performer
```

Comment:
This should have the full type `/mob/living/carbon/human/var/mob/living/carbon/human/cpr_performer` and be cleared in `/mob/living/carbon/human/Destroy` to avoid even more cleanup problems.

Alternatively, do you really need to bother with all that ? There's probably simpler ways to go about this
For example, you could set a "CPR Start" cool down, preventing anyone from doing it during same duration. if do_after fails, clear it to allow someone else to.

Reply:
I've changed it to use a timer now instead, does it look fine to you now?